presubmit-service/check_if_pr_can_cherry_pick.sh

```diff
@@ -0,0 +1,66 @@
+#! /bin/bash
+
+# The organization containing the repo to be checked against, e.g. angular
+ORG=$1
```

Comment:
Use snake_case for bash variables

(CAPS_CASE should only be used for environment variables)